pubsub/query/parser_test.go

```diff
@@ -83,9 +83,9 @@ func TestParser(t *testing.T) {
 	for _, c := range cases {
 		_, err := query.New(c.query)
 		if c.valid {
-			assert.NoError(t, err, "Query was '%s'", c.query)
+			assert.NoErrorf(t, err, "Query was '%s'", c.query)
```

Comment:
not a real method. same below

Reply:
hard for me to understand why `make` and then tests seem to pass but i believe you @ebuchman

Reply:
https://github.com/stretchr/testify/blob/master/assert/assertion_format.go#L236

Reply:
what triggered the change? `assert.NoErrorf(t, err, "Query was '%s'", c.query)` this should also compile, no?

Reply:
this `go vet` error @melekes   ```	./...
WARNING: staticcheck, gosimple and unused are all set, using megacheck instead
pubsub/query/parser_test.go:88::error: possible formatting directive in Error call (vet)
``` goes away when adding `f` at the end similar to @odeke-em solutions elsewhere i believe